docs/manuals/user/using.md

```diff
@@ -297,6 +297,20 @@ in any of the attribute metadata section in the notification.
 As specified in the
 [FIWARE data models documentation](https://fiware-datamodels.readthedocs.io/en/latest/guidelines/index.html#dynamic-attributes).
 
+1. **observedAt** NGSI-LD metadata. The most recent `observedAt` attribute value
+found in any of the attributes in the notification.
+As specified in the
+[NGSI-LD](https://github.com/smart-data-models/data-models/blob/master/ngsi-ld_howto.md#steps-to-migrate-to-json-ld).
```

Comment:
@jason-fox any better reference than this?

Reply:
Direct from the [NGSI-LD Specification](https://www.etsi.org/deliver/etsi_gs/CIM/001_099/009/01.04.01_60/gs_cim009v010401p.pdf)

>
> In case of conflicting information for an Attribute, where a `datasetId` is duplicated, but there are differences in the other attribute data, the one with the most recent `observedAt` DateTime, if present, and otherwise the one with the most recent `modifiedAt` DateTime shall be provided. 

Strictly this quote is about multi-attributes, but the precedence order is always the same

Reply:
i was hoping for a web spec, but that's life :)

Reply:
also, it seems from `https://ngsi-ld-tutorials.readthedocs.io/en/latest/ngsi-ld-operations.html` that `modifiedAt` can be attached to entity itself and not just the attribute (despite from the examples it seems that this value would be the same as the last modification of an attribute).

Reply:
This is `options=sysAttrs` - the `modifiedAt` is generated by the Context Broker for each attribute and for the entity as a whole. I think there could be a divergence if an entity has multiple attributes and only some are updated since the unmodified attributes would have an older `modifiedAt` timestamp.

Obviously this won't be an issue for a subscribing component since the notification will only be raised when an attribute changes.

Reply:
@jason-fox thanks! is this documented in any official doc of ngsi-ld or a related implementation? I think it would be good to refer to it in the internal ql docs